tox.ini

```diff
@@ -51,6 +51,15 @@ basepython = python2.6
 deps = {[testenv]basedeps}
        django>=1.5,<1.6
 
+[testenv:gae]
```

Comment:
If not, it'll die with "dev_appserver.py" not found. I'm not sure if there's another way to surface this.